### PR TITLE
Proposed work mode for charter development

### PR DESCRIPTION
**Pull request by brentzundel**
Signed-off-by: Brent Zundel <brent.zundel@gmail.com>

### CODE REVIEW

**Review by iherman — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,3 +23,18 @@ If you added a contributor by mistake, you can remove them in a comment with:
...
+the listed chairs before merging. Other approvals may be sought, but are not
+required, particularly while this document is in a draft state.
+
+PRs will be merged at the chairs' discretion, and may be merged quickly. Please
+raise an issue if changes have been made to the draft charter that you do not
+agree with. However, please keep in mind that while we will strive to address
+issues in a manner that is acceptable to those have raised them, we cannot
+guarantee that all issues will result in changes satisfactory to all parties,
+nor that all parties will have sufficient time to review changes before they are
+merged into the draft.
```
Comment (msporny) [inline CONTRIBUTING.md]: > PRs will be merged at the chairs' discretion, and may be merged quickly. 

The VCWG Charter is a VCWG work item, isn't it? Why are we not allowing VCWG members enough time to review PRs? This creates an atmosphere where things can be rushed through, which could then just inevitably lead to objections when the charter goes out to review.

My suggestion is that PRs that are not Editorial in nature must be given time on at least one call for discussion, and a minimum of 7 days from the day they were raised, to be merged by the Chairs.

Is there some kind of deadline that we're working against that is driving this sort of work mode?

**Review by msporny — CHANGES_REQUESTED** [review]
The VCWG Charter is a VCWG work item, isn't it? Why are we not ensuring that VCWG members have enough time to review PRs to a charter to a WG that they will be participating in?

This creates an atmosphere where things can be rushed through, which could then just inevitably lead to objections when the charter goes out to review. I don't understand the rush here, what did I miss?

My suggestion is that PRs must be given time on at least one call for discussion, and a minimum of 7 days from the day they were raised, to be merged by the Chairs.

File: CONTRIBUTING.md (hunk 1 above)
Comment (brentzundel) [reply CONTRIBUTING.md]: I'd like to see the charter draft ready for AC review within a couple of weeks, which imo doesn't allow for review periods, 
I have added a line which I hope allays concerns that the chairs will submit a charter which does not meet with the approval to the AC for review.

**Review by msporny — APPROVED** [review]
> Before the draft charter is sent for Advisory Committee review it will be ￼submitted to the current VCWG for approval.

I'm now ok w/ this work mode as long as the VCWG reaches consensus on the final charter before submission to the AC for voting.

**Comment by brentzundel** [comment]
@Sakurann I am hoping to get your opinion on this proposed work mode.

**Review by kdenhartog — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]
I think this Work Mode makes sense. Thank you, Brent.

**Pull request merged**